source/hp/dof_handler.cc

```diff
@@ -1863,7 +1881,7 @@ namespace hp
   {
     // Create sufficiently many hp::DoFLevels.
     while (levels.size() < tria->n_levels())
-      levels.emplace_back(new dealii::internal::hp::DoFLevel);
+      levels.emplace_back(new internal::hp::DoFLevel);
```

Comment:
Do you need this change? I bet that there is some version of Microsoft Visual Studio out there that requires this :-( Same in line 1962 (new)

Reply:
Ah, I forgot about that :(
I'll revert these changes.